src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -747,23 +753,19 @@ static void verify(ModuleSymbol m)
 
     /// <summary>
     /// Demonstrates that modopt encoding of 'ref readonly' parameters in function pointers
-    /// won't break older compilers (they will see the parameter as 'in').
+    /// won't break older compilers (they will see the parameter as 'ref').
     /// </summary>
     [Fact]
     public void FunctionPointer_Modopt_CustomAttribute()
```

Comment:
>FunctionPointer_Modopt_CustomAttribute [](http://example.com/codeflow?start=16&length=38)

I do not get why the test was modified. Consider keeping the original scenario. If behavior changes, we would like to clearly see that during review. Add a new test for new scenarios.